src/types/Plan.ts

```diff
@@ -40,3 +42,8 @@ export enum PlanLevel {
     AGENCY_LARGE = 'agency_large',
     AGENCY_HUGE = 'agency_huge',
 }
+
+export enum PlanBillingCycle {
```

Comment:
There is a `SignupBillingCycle` already. Not sure if we should just define a general `BillingCycle` type to use in both?